execute/window_test.go

```diff
@@ -105,21 +105,45 @@ func TestWindow_GetEarliestBounds(t *testing.T) {
 		{
 			name: "simple with offset",
 			w: MustWindow(
-				values.ConvertDuration(5*time.Minute),
-				values.ConvertDuration(5*time.Minute),
-				values.ConvertDuration(30*time.Second)),
+				values.ConvertDurationNsecs(5*time.Minute),
+				values.ConvertDurationNsecs(5*time.Minute),
+				values.ConvertDurationNsecs(30*time.Second)),
 			t: execute.Time(5 * time.Minute),
 			want: execute.Bounds{
 				Start: execute.Time(30 * time.Second),
 				Stop:  execute.Time(5*time.Minute + 30*time.Second),
 			},
 		},
+		{
+			name: "simple months",
+			w: MustWindow(
+				values.ConvertDurationMonths(5),
+				values.ConvertDurationMonths(5),
+				values.ConvertDurationMonths(0)),
+			t: values.ConvertTime(time.Date(1970, time.January, 1, 0, 0, 0, 0, time.UTC)),
+			want: execute.Bounds{
+				Start: values.ConvertTime(time.Date(1970, time.January, 1, 0, 0, 0, 0, time.UTC)),
+				Stop:  values.ConvertTime(time.Date(1970, time.June, 1, 0, 0, 0, 0, time.UTC)),
+			},
+		},
+		{
+			name: "simple months with offset",
+			w: MustWindow(
+				values.ConvertDurationMonths(5),
+				values.ConvertDurationMonths(5),
+				values.ConvertDurationMonths(5)),
+			t: values.ConvertTime(time.Date(1970, time.January, 1, 0, 0, 0, 0, time.UTC)),
+			want: execute.Bounds{
+				Start: values.ConvertTime(time.Date(1970, time.January, 2, 4, 30, 0, 0, time.UTC)),
+				Stop:  values.ConvertTime(time.Date(1970, time.June, 2, 4, 30, 0, 0, time.UTC)),
+			},
```

Comment:
Can you explain why it outputs these bounds? Maybe I don't understand how monthly offsets work, but I would have expected a window with period equal to offset to be the same as a window with that same period and no offset.

Reply:
Whoa, sorry I thought I fixed that. Should be fixed now.